components/link-icon/template.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <title>Web component preview</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Reusable preview HTML page for testing components">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="crossorigin">
+    <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@400;700&display=swap" rel="stylesheet">
+    <style>
+      body {
+        font-family: 'Public Sans', sans-serif;
+        font-weight: 500;
+      }
+    </style>
+    <link rel="stylesheet" href="index.css">
+    <script type="module">
+      import './src/index.js';
+    </script>
+  </head>
+
+  <body>
+
+    <div class="main">
+      <a href="link.pdf">This is pdf link</a>
+      <a href="google.html">External link</a>
+    </div>
```

Comment:
I had some trouble viewing/testing this component locally. I traced the problem to this snippet in `template.html`. With the following change, everything works as expected.

```html
<main>
    <a href="link.pdf">This is pdf link.</a>
    <a href="https://google.com">This is an external link.</a>
    <a href="google.html">This is an internal link.</a>
</main>
```

So it wasn't a problem with any of the JS, just this preview HTML.